pymeasure/instruments/rohdeschwarz/sfm.py

```diff
@@ -0,0 +1,1501 @@
+        if subsystem is None:
+            self.write("CAL:MOD%d" % (number))
+        else:
+            self.write("CAL:MOD%d:%s" % (number,strict_discrete_set(subsystem,
+                                        ["NIC","VIS","SOUN1","SOUN2","COD"])))
+
+    #INST (Manual 3.6.4)
+    system_number = Instrument.control(
+        "INST:SEL?",
+        "INST:SEL:%s",
+        """A int property for the selected systems (if more then 1 available)
+
+        * Minimum 1
+        * Maximum 6
+
+        """,
+        validator = strict_discrete_set,
+        values = {1:"SYS1",2:"SYS2",3:"SYS3",4:"SYS4",5:"SYS5",6:"SYS6"},
+        map_values=True,
+        check_set_errors=True,
+        )
+
+    #ROUTE (Manual 3.6.5)
+    #TODO: CHAN-->CHAN[1..6]
+    R75_out = Instrument.control(
+        "ROUT:CHAN:OUTP:IMP?",
+        "ROUT:CHAN:OUTP:IMP %s",
+        """ A bool property that controls the use of the 75R output (if installed)
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   50R output active (N)
+        True    75R output active (BNC)
+        ======  =======
+        """,
+        validator = strict_discrete_set,
+        values={False:"LOW", True:"HIGH"},
+        map_values = True,
+        )
+
+    ext_ref_basic = Instrument.control(
+        "ROUT:REF:CLOCK:BAS?",
+        "ROUT:REF:CLOCK:BAS %s",
+        """ A bool property for the external reference for the basic unit
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   Internal 10 MHz is used
+        True    External 10 MHz is used
+        ======  =======
+
+        """,
+        validator = strict_discrete_set,
+        values={False:"INT", True:"EXT"},
+        map_values = True,
+        )
+
+    ext_ref_extenstion = Instrument.control(
+        "ROUT:REF:CLOCK:EXT?",
+        "ROUT:REF:CLOCK:EXT %s",
+        """ A bool property for the external reference for the extension frame
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        False   Internal 10 MHz is used
+        True    External 10 MHz is used
+        ======  =======
+        """,
+        validator = strict_discrete_set,
+        values={False:"INT", True:"EXT"},
+        map_values = True,
+        )
+    #TODO: VID-->VID[1..6]
+    ext_vid_connector = Instrument.control(
+        "ROUT:TEL:VID:EXT?",
+        "ROUT:TEL:VID:EXT %s",
+        """A string property controlling which connector is used as the input of the video source
+
+        Possible selections are:
+
+        ======  =======
+        Value   Meaning
+        ======  =======
+        HIGH    Front connector - Hi-Z
+        LOW     Front connector - 75R
+        REAR1   Rear connector 1
+        REAR2   Rear connector 2
+        AUTO    Automatic assignment
+        ======  =======
+        """,
+        validator = strict_discrete_set,
+        values = ["HIGH","LOW","REAR1","REAR2","AUTO"],
+        )
+
+    #Frequency (3.6.6.1)
```

Comment:
Exactly, these are markers to the section of the manual the following block is covered.

I will follow your proposal and make these references a sentence in the first property of the regarding subsystem.

In this case it will look like this:
```    ...
        ======  =======

        refer also to chapter 3.6.6.1 of the manual
        """,
```